### ISSUE TITLE
[Feature Request] any chance to add Accessibility Information to `hs`'s status item?

### ISSUE THREAD

**Issue by godbout** [Hammerspoon/hammerspoon#3288]
that would help reaching the menu through different means.

i can definitely do a PR if this would get merged. let me know! thanks.

**Comment by godbout** [comment]
forgot the:

<img width="581" alt="Screenshot 2022-09-08 at 14 52 50" src="https://user-images.githubusercontent.com/121373/189055007-1aa12d07-a59e-4125-85ab-87e30f14b3b1.png">

**Comment by latenitefilms** [comment]
Not sure I entirely understand what you're asking... but you should be able to use [`hs.axuielement`](http://www.hammerspoon.org/docs/hs.axuielement.html) to get what you want?

**Comment by godbout** [comment]
@latenitefilms my bad, let me try to be more clear:

i build an app that reaches the macOS UI through reading the Accessibility information of UI Elements. see below:

https://user-images.githubusercontent.com/121373/190856291-c1852d90-e1b8-4fdd-8510-de8c77c49915.mp4

at the end of the video, you can see that the `Hammerspoon` status bar icon is not reachable. the reason is because `Hammerspoon` doesn't have any Accessibility Labels set, in `Hammerspoon`'s source code.

adding Accessibility information in one's status bar icon is as easy as:
```
        statusItem.button?.setAccessibilityLabel("some label")
        statusItem.button?.setAccessibilityTitle("some title")
        statusItem.button?.setAccessibilityHelp("some help")
```
see https://developer.apple.com/documentation/appkit/nsaccessibilityprotocol/1529606-setaccessibilitylabel.

it's clearly seeable that the `Hammerspoon` status bar icon doesn't have any info, by using the Accessibility Inspector that comes with Xcode. see below:

https://user-images.githubusercontent.com/121373/190856472-5a8351f4-c89b-4a5c-b8a5-8c5c209cf35f.mp4

i guess the Accessibility could/should be set somewhere around here: https://github.com/Hammerspoon/hammerspoon/blob/1bd0c184a6b2acbffad27498b9bb15af3e116b8b/Hammerspoon/MJMenuIcon.m#L30

hope this clarifies!

**Comment by godbout** [comment]
> but you should be able to use [`hs.axuielement`](http://www.hammerspoon.org/docs/hs.axuielement.html) to get what you want?

so my point is not using hs' API. it's about making hs' own menu icon reachable through the macOS AX API.
